packages/color-block/src/components/list/ListItem.tsx

```diff
@@ -0,0 +1,141 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import {
+    Badge,
+    BadgeEmphasis,
+    Button,
+    ButtonStyle,
+    Color,
+    IconSize,
+    IconTrashBin,
+    Tooltip,
+    TooltipPosition,
+    useCopy,
+} from '@frontify/fondue';
+import { joinClassNames } from '@frontify/guideline-blocks-shared';
+
+import { ColorName } from '../ColorName';
+import { ColorPickerFlyout } from '../ColorPickerFlyout';
+import { TooltipContent } from '../TooltipContent';
+import { mapColorSpaces } from '../../helpers/mapColorSpaces';
+import { ColorBlockType, ItemProps } from '../../types';
+import { ColorSpaceValue } from '../ColorSpaceValue';
+
+export const ListItem = ({ color, colorSpaces, isEditing, onBlur, onUpdate, onDelete }: ItemProps) => {
+    const { copy, status } = useCopy();
+
+    const mappedFirstColorSpace = mapColorSpaces(colorSpaces[0], color);
+
+    return (
+        <div
+            data-test-id="list-item"
+            className="tw-group tw-relative tw-flex tw-shadow-t-inner-line tw-transition-all hover:tw-shadow-t-inner-line-strong"
+        >
+            {!isEditing ? (
+                <Tooltip
+                    withArrow
+                    position={TooltipPosition.Right}
+                    hoverDelay={0}
+                    content={<TooltipContent colorValue={mappedFirstColorSpace.value ?? ''} status={status} />}
+                    triggerElement={
+                        <div className="tw-h-full tw-mr-9 tw-bg-[url('https://cdn.frontify.com/img/transparent.png')] tw-bg-[length:10px_10px]">
+                            <div
+                                data-test-id="color-tooltip-trigger"
+                                className="tw-w-[120px] tw-h-full tw-min-h-[60px] tw-cursor-pointer tw-shadow-t-inner-line tw-transition-all group-hover:tw-shadow-t-inner-line-strong"
+                                style={{
+                                    backgroundColor: `#${color.hex}`,
```

Comment:
reuse style: maybe even add to helper file, as it is the same as in the other file

Reply:
Ideally, this would be returned from the backend with the `#`, so I hope this will be changed in the future and make this transform obsolete

Reply:
That would even be better 👍 :D